pkgs/build-support/vm/default.nix

```diff
@@ -90,6 +90,10 @@ rec {
     done
 
     mount -t devtmpfs devtmpfs /dev
+    ln -s /proc/self/fd /dev/fd
```

Comment:
The following devices might be also helpful: `/dev/null`, `/dev/full`, `/dev/random`, `/dev/tty`, `/dev/urandom`, `/dev/zero` and maybe also some filesystems like `/dev/shm`, `/dev/pts`

Reply:
Not sure what is currently available.

Reply:
These are all available.